SCClassLibrary/Common/Collections/SequenceableCollection.sc

```diff
@@ -64,6 +64,16 @@ SequenceableCollection : Collection {
 		});
 		^obj
 	}
+	*recursiveFormula { arg size, a=0.0;
+		var i=0, divisor = 3, divisorDec = 0.9;
+		var obj = this.new(size);
+		while { i < size }{
+			obj.multiply((a/divisor));
+			divisorDec -= 0.1;
```

Comment:
Operators like `+=`, `-=`, `*=`, `/=` don't exist in sclang. The only correct way to decrement a number by 0.1 would be
`number = number - 0.1;`
Have you ever tested your method before making a pull request?